cookiecutter.json

```diff
@@ -46,7 +46,6 @@
   ],
   "use_async": "n",
   "use_drf": "n",
-  "custom_bootstrap_compilation": "n",
```

Comment:
Is it that we can't easily provide a custom Bootstrap compliation anymore or is it always enabled?